egasub/submission/submittable/base.py

```diff
@@ -32,6 +32,18 @@ def metadata(self):
     def status(self):
         return self._status
 
+    @property
+    def local_validation_errors(self):
+        return self._local_validation_errors
+
+    def _add_local_validation_errors(self, type_, alias, field, message):
```

Comment:
should rename this to `_add_local_validation_error`, ie, `error` in singular form.